pkg/crc/machine/machine.go

```diff
@@ -830,3 +835,22 @@ func configProxyForCluster(ocConfig oc.OcConfig, sshRunner *crcssh.SSHRunner, sd
 
 	return nil
 }
+
+func checkProxyPropagatedToCluster(ocConfig oc.OcConfig, proxyConfig *network.ProxyConfig) {
+	checkProxySettingsForOperator := func() error {
+		proxySet, err := cluster.CheckProxySettingsForOperator(ocConfig, proxyConfig, "redhat-operators", "openshift-marketplace")
+		if err != nil {
+			logging.Debugf("Error getting proxy setting for openshift-marketplace operator %v", err)
+			return &errors.RetriableError{Err: err}
+		}
+		if !proxySet {
+			logging.Debug("Proxy changes for cluster in progress")
+			return &errors.RetriableError{Err: fmt.Errorf("")}
+		}
+		return nil
+	}
+
+	if err := errors.RetryAfter(60, checkProxySettingsForOperator, 2*time.Second); err != nil {
+		logging.Debug("Proxy settings have not been applied yet to cluster")
```

Comment:
Sounds a bit like "the settings are not applied yet, but they are going to be applied later", which we don't know as we just failed to apply them. I would reword this to be less optismistic "Failed to propagate proxy settings to cluster" or something like this.